ports/raspberrypi/boards/wk-50/mpconfigboard.mk

```diff
@@ -0,0 +1,12 @@
+USB_VID = 0xFEE5
+USB_PID = 0x6036
```

Comment:
Where did you get this? It doesn't look legit. You may want to request one from pid.codes.

Reply:
That isn't valid. I don't know how they came up with it. Let's not propagate it.